android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModel.kt

```diff
@@ -155,10 +156,15 @@ constructor(
                       questionnaireConfig.interpolate(
                         event.resourceData?.computedValuesMap ?: emptyMap()
                       )
+                    var groupResourceId: String? = null
+
                     val params =
                       actionConfig
                         .params
                         .map {
+                          if (it.key == "familyLogicalId") {
+                            groupResourceId = it.value
+                          }
```

Comment:
This means the config authors have to name the rules a certain way. I don't think this is the correct implementation. 
- We can't control how people name their rules while authoring configs.

Reply:
Why is this hardcoded?